src/app/containers/Brand/index.test.jsx

```diff
@@ -42,5 +61,20 @@ describe(`BrandContainer`, () => {
 
       expect(skipLinkHref).toBe('#content');
     });
+
+    it('should update cookie to preffered variant when ScriptLink is clicked', () => {
```

Comment:
Do we also need a test for when a preference cookie has already been set?

Reply:
Added a test for when a preference cookie already exists.